app/src/common/shared/org/mozilla/vrbrowser/VRBrowserActivity.java

```diff
@@ -344,6 +348,8 @@ protected void onResume() {
 
     @Override
     protected void onDestroy() {
+        TelemetryWrapper.destroy();
+
```

Comment:
I am curious why you want to queue your multiple window telemetry at `onDestory()` instead of be like our previous telemetry, queue them at `onStart()` and `onStop()`?

Reply:
Some windows related events like window lifetime histogram or window placement pct are better measured at the end of the session as you can stop and resume but the windows are not recreated so the stat wouldn't be completely accurate if we flush the data in the middle. We could artificially trigger a window destroy/re-create at onStop but this just made things simpler.

Reply:
put in onDestroy, I would be afraid the metric will not be uploaded for Oculus release. We usually do it `onStart/onStop` for `recordSessionStart/recordSessionEnd` and then scheduleUpload().

IIUC, `onDestroy ` will be called after `onStop`?

Reply:
Fixed this